docs/SetupGuide_Dotnet.md

```diff
@@ -399,6 +401,33 @@ public static IActionResult Run(
     return new CreatedResult($"/api/addproduct", product);
 }
 ```
+## CSharp Scripting
```

Comment:
Let's move this under `Setup Function Project` and have two subsections there - one for traditional .NET and the other for CSX

Reply:
subsections for just setup instructions or for everything we follow that pattern?

Reply:
Everything where there's differences between the two

Reply:
Right now the way it's structured will be confusing I think because users will come in and start at the top which tells them how to create a normal .NET function project - and only when they finally get down here will they see what to do for C# scripting.

Instead I think it would be clearer to just have each section with differences for the two types to have a subsection for each instead (in the case of the samples probably fine just to have two links in each since it's just one line)

Reply:
okay after contemplating on how to structure this, I felt it would be way cleaner to have a separate file for csx and added it.